specifications/api/swagger.yaml

```diff
@@ -4324,13 +4407,36 @@ paths:
                   title: "redemption"
       responses: *responsesPostTransactionFee
 
+  /wallets/{walletId}/transactions/sign:
+    post:
+      operationId: postSignTransaction
+      tags: ["Transactions"]
+      summary: Sign
+      description: |
+        <p align="right">status: <strong>under development</strong></p>
+
+        Sign a transaction body and return the full transaction. That
+        is, create witnesses using the keys available to this wallet
+        and attach them to the body.
+
+        If a full transaction is passed in the request, then witness
+        will be added to the existing witness set.
+      parameters:
+        - *parametersWalletId
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema: *ApiPostSignTransactionData
+      responses: *responsesPostSignedTransaction
+
   /wallets/{walletId}/transactions:
     post:
       operationId: postTransaction
-      tags: ["Transactions"]
+      tags: ["Legacy Transactions"]
```

Comment:
what about waiting with `Transactions -> Legacy Transactions` and `stable -> deprecated` until the last ticket in this epic?

Reply:
OK fair enough.